modules/ROOT/pages/x12-receive-read-settings.adoc

```diff
@@ -1,148 +1,162 @@
-|Disabled
+|Fail when invalid character in value
+a| * If selected, the host rejects transactions with invalid characters.
+* If unselected, the characters are either passed through or substituted.
 
-|Fail document if value is repeated too many times
-|If selected, a transaction with values that are repeated too many or too few times are rejected. If unselected, the values are accepted and the transaction is not rejected. In either case, the error is reported in the 997 acknowledgments.
 |Enabled
 
-|Fail document if unknown segments are used
-|If selected, a transaction containing any unknown segments is rejected. If unselected, unknown segments are ignored and the transaction is not rejected. In either case, the error is reported in the 997 acknowledgments.
+|Fail if value is repeated too many times
+a|* If selected, the host rejects transactions with values that are repeated too many or too few times.
+*Vijay, is the "too few times" correct? It is in the current text.*
+* If unselected, the values are accepted.
 |Enabled
 
-|Fail document when segments are out of order
-|If selected, a transaction with segments that are out of order is rejected. If unselected, the segment can be reordered the transaction is not rejected. In either case, the error is reported in the 997 or 999 acknowledgments.
-|Enabled
+|Fail if unknown segments are used
+a|* If selected, the host rejects transactions that contain unknown segments.
+* If unselected, If unselected, the host ignores unused segments.
+|Disabled
 
-|Fail document when too many repeats of a segment
-|A transaction with a segment repeated too many times is rejected. The error is reported in the 997 or 999 acknowledgments.
-|Enabled
+|Fail when segments are out of order
+a|* If selected, the host rejects transactions that have out-of-order segments.
+* If unselected, the segments are reordered.
+| Enabled
 
-|Fail document when unused segments are included
-|If selected, a transaction containing segments marked as unused in the schema is rejected. If unselected, the transaction is not rejected and the unused segments are ignored. In either case, the error is reported in the 997 or 999 acknowledgments.
+|Fail when too many repeats of segment
+| If selected, the host rejects transactions with segments repeated too many times.
 |Enabled
+
+|Fail when unused segments are included
+a|* If selected, the host rejects transactions containing segments marked as unused in the schema.
+* If unselected, the host ignores unused segments.
++
+*Vijay, what is the difference between this option and the "Fail if unknown segments are used* setting? Both are in the interface. Is this a bug?*
+| Disabled
+
+|Enforce conditional relationship validations a| * If selected, any missing conditional data elements causes the generation of the X12 transaction to fail.
+* *Vijay, what happens if this is not enabled? Is the X12 transaction generated with out-of-order segments*?
```

Comment:
@dhirsch1 - When it is not enabled, we do not enforce the conditional relationship validation.
An example of the conditional relationship is - If the EDI data is specifying, Weight value in an element, there will be also an associated qualifier element that is used to specify the Unit of Measurement (just an example, but there are other similar cases as well)

If weight value is presented, then the Unit of Measurement (LB, Kg etc) also must be present. 
With this validation turned ON - in such elements that are defined with conditional relationship - when one value does not exist - the transaction is marked as failed.
With the validation turned OFF - even if we get only weight value, but no Unit of Measurement - the transaction still goes through without failure.